pkg/scheduler/plugins/nodenumaresource/topology_eventhandler.go

```diff
@@ -118,14 +119,29 @@ func (m *nodeResourceTopologyEventHandler) updateNodeResourceTopology(oldNodeRes
 		}
 	}
 
+	// remove cpus reserved by node.annotation.
+	var CPUsReservedByNode cpuset.CPUSet
```

Comment:
Here we can optimize the implements.
1. There are so many lines to parse the `node reservation` annotation. We can move the codes as functions in `apis/extension/node_reservation.go`, so we can reuse the functions as we need.
2. If the users just reserved CPU quantity like `cpu: 10`, so which specific CPU cores should be reserved? For the two features of CPU Suppress in koordlet and NodeNUMAResource in koord-scheduler, it is necessary to specify that the specific CPU core is reserved. Just like the kubelet cpu manager policy `static` with reservation.